server/api/src/main/scala/com/prisma/api/database/DatabaseMutationBuilder.scala

```diff
@@ -297,26 +319,19 @@ object DatabaseMutationBuilder {
     } yield action
   }
 
-  def triggerFailureWhenNotExists(project: Project, query: SQLActionBuilder, table: String) = {
+  def triggerFailureWhenExists(project: Project, query: SQLActionBuilder, table: String, notExists: Boolean = false) = {
```

Comment:
We should try to avoid boolean in public signature. I think we should rather two method names, e.g. `triggerFailureWhenExists` and `triggerFailureWhenNotExists`.

Reply:
Ah, now i just saw exactly that below. I would do two things to avoid this confusion:
1. Move `triggerFailureWhenNotExists` to allow reading from top to bottom.
2. Add a private method `triggerFailureWhenInternal` that takes this boolean. So there will be 3 methods: `triggerFailureWhenNotExists`, `triggerFailureWhenExists`, `triggerFailureWhenInternal`. And only the private one contains the complex boolean.

Reply:
Done.